packages/paypal-js/src/utils.ts

```diff
@@ -158,3 +158,5 @@ function createScriptElement(
 
     return newScript;
 }
+
+export const isServer = typeof window === "undefined";
```

Comment:
Two minor things:

1. should we make this a function so its callable in different contexts?
2. Can we also check `document`? When doing server-side rendering, I think some environments may set `window` so its safer to check `document`. Might as well check for both being defined.

```suggestion
export function isServer() { 
  return typeof window === "undefined" && typeof document === "undefined";
}
```